flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_NavRadioPage.js

```diff
@@ -114,20 +114,14 @@ class CDUNavRadioPage {
             };
             ilsFrequencyCell = "[\xa0\xa0]/[\xa0\xa0.\xa0]";
             ilsCourseCell = "";
-            const approach = mcdu.flightPlanManager.getApproach();
-            const ilsIdent = mcdu.radioNav.getILSBeacon(1);
-            const runway = mcdu.flightPlanManager.getApproachRunway();
             if (mcdu.ilsFrequency != 0) {
-                let ilsIdentStr = "[\xa0\xa0]";
-                if (ilsIdent.ident != "") {
-                    ilsIdentStr = "{small}" + ilsIdent.ident + "{end}";
-                }
                 if (mcdu._ilsFrequencyPilotEntered) {
-                    ilsFrequencyCell = ilsIdentStr + "/" + mcdu.ilsFrequency.toFixed(2);
-                    ilsCourseCell = "{inop}____{end}";
-                } else {
-                    ilsFrequencyCell = ilsIdentStr + "{small}" + "/" + mcdu.ilsFrequency.toFixed(2) + "{end}";
-                    ilsCourseCell = "{small}" + ilsIdent.course.toFixed(0).padStart(3, "0") + "{end}";
+                    const ilsIdent = mcdu.radioNav.getILSBeacon(1);
+                    ilsFrequencyCell = `{small}${ilsIdent.ident.trim().padStart(4, "\xa0")}{end}/${mcdu.ilsFrequency.toFixed(2)}`;
+                    ilsCourseCell = "{small}F" + ilsIdent.course.toFixed(0).padStart(3, "0") + "{end}";
+                } else if (mcdu.ilsAutoTuned) {
```

Comment:
Is there a scenario where `mcdu.ilsFrequency` is non-zero, yet both `_ilsFrequencyPilotEntered` and `ilsAutoTuned` are false-ish? Aren't they somehow exclusive?

Reply:
I don't think there is, but I do prefer an explicit condition, to protect against future changes (e.g. somebody adds a bit of code to sync mcdu.ilsFrequency with the sim on spawn)... that said, this is all ripped out and replaced in my upcoming nav radio branch so it won't be around long anyway... Can I leave it as-is for now?